.gitignore

```diff
@@ -1 +1,2 @@
 codecept.phar
+node_modules
```

Comment:
I think that adding `node_modules` to ignored folders here is not a bad idea, though I'm not sure what might be the use case for having `node_modules` directly inside the `dev-lib`.